conda/meta.yaml

```diff
@@ -11,71 +11,71 @@ source:
 
 requirements:
   build:
-    - armadillo>=9.900.5
+    - armadillo=12.8.4
     - boost=1.80.0
-    - cmake>=3.21.3
+    - cmake=3.25.1
     - cuda-libraries-dev=12.3.0                  # [linux64]
     - cuda-libraries=12.3.0                      # [linux64]
     - cuda-runtime=12.3.0                        # [linux64]
     - cuda-nvcc=12.3.52                          # [linux64]
     - dcmtk=3.6.1
     - fftw=3.3.9
     - gcc_linux-64=9.4.0                         # [linux64]
-    - gadgetron::gadgetron-python>=1.4.1
-    - gmock>=1.10.0
+    - gadgetron-python>=1.4.1
+    - gmock=1.14.0
     - gxx_linux-64=9.4.0                         # [linux64]
-    - gtest=1.10.0
-    - h5py=3.2.1
+    - gtest=1.14.0
+    - h5py=3.7.0
     - hdf5=1.10.6
     - howardhinnant_date=3.0.1
-    - ismrmrd::ismrmrd=1.13.7
-    - ismrmrd::ismrmrd-python>=1.9.8
+    - ismrmrd=1.13.7
```

Comment:
Channel specifiers are missing here too, might be fine by now, but there were a lot of versions floating around at some point.

Reply:
It is ok now. For future, all ismrmrd should only be published as conda ? If so, it is safe for future versions, without the channel spec.